dashboard/src/app/workspaces/workspace-details/warnings/workspace-warnings.controller.ts

```diff
@@ -33,9 +33,6 @@ export class WorkspaceWarningsController {
     this.warnings = [];
 
     let environment = (this.workspace && this.workspace.config && this.workspace.config.defaultEnv) ? this.workspace.config.environments[this.workspace.config.defaultEnv] : null;
-    if (environment) {
-      this.warnings = this.warnings.concat(environment.warnings);
-    }
 
     if (this.workspace && this.workspace.runtime) {
       this.warnings = this.warnings.concat(this.workspace.runtime.warnings);
```

Comment:
do we still have warnings in workspace.runtime ?

Reply:
yes

Reply:
then environment variable could be removed https://github.com/eclipse/che/pull/7663/files#diff-96c3dcf163c4af0310e5640e7e88ecc4L35

Reply:
and mentioning about environment in comments https://github.com/sleshchenko/che/blob/5f2f76bb9a54abd543eb9df3b7e007ddd5ed9e23/dashboard/src/app/workspaces/workspace-details/warnings/workspace-warnings.controller.ts#L15